seqr/views/utils/dataset_utils.py

```diff
@@ -295,26 +294,106 @@ def match_and_update_samples(
 
     return samples, matched_individual_ids, activated_sample_guids, inactivated_sample_guids, family_guids_to_update, remaining_sample_ids
 
-
 def _parse_tsv_row(row):
     return [s.strip().strip('"') for s in row.rstrip('\n').split('\t')]
 
+RNA_OUTLIER_COLUMNS = {'geneID': 'gene_id', 'pValue': 'p_value', 'padjust': 'p_adjust', 'zScore': 'z_score'}
+
+SAMPLE_ID_COL = 'sample_id'
+GENE_ID_COL = 'gene_id'
+TPM_COL = 'TPM'
+TISSUE_COL = 'tissue'
+INDIV_ID_COL = 'individual_id'
+TPM_HEADER_COLS = [SAMPLE_ID_COL, GENE_ID_COL, TISSUE_COL, TPM_COL]
+
+TISSUE_TYPE_MAP = {
```

Comment:
Instead of having a hard-coded list of tissues here, it may be worth treating tissue-type as an opaque label, or even better - allowing any GTEx tissue and adding docs so people uploading data know this. GTEx metadata in this file 
https://storage.googleapis.com/gtex_analysis_v8/annotations/GTEx_Analysis_v8_Annotations_SampleAttributesDS.txt
has an SMTS column with tissue labels for all GTEx samples:

```
   1327 Adipose Tissue
    275 Adrenal Gland
     21 Bladder
   3480 Blood
   1473 Blood Vessel
    217 Bone Marrow
   3326 Brain
    480 Breast
     19 Cervix Uteri
    821 Colon
   1582 Esophagus
      9 Fallopian Tube
   1141 Heart
    104 Kidney
    251 Liver
    867 Lung
   1132 Muscle
    722 Nerve
    195 Ovary
    360 Pancreas
    301 Pituitary
    262 Prostate
    181 Salivary Gland
   2014 Skin
    193 Small Intestine
    260 Spleen
    381 Stomach
    406 Testis
    812 Thyroid
    166 Uterus
    173 Vagina
```

Reply:
This behavioris perfectly sufficient for our current use case so I see no need to change it now, especially given that the tissue type is a database enumeration which would require migrations to change. If in the future we get new tissue types and need to update the code, it may be worth removing the database enumeration and using this approach instead, but I would want to make that decision when it was actually relevent